### ISSUE TITLE
Issue with NginX and Letsencrypt

### ISSUE THREAD

**Issue by Intelioum** [apioo/fusio#389]
Hi 
I was looking for an easy solution to create a simple API on top od MS SQL Server and came across fusio. I am impressed with the ease of use and environment but when I tried to set it up for a real world scenario, it seemed that documentation is not complete or helpful.
with http everything is fine but I have problem setting it up with SSL Cert from Letsencrypt, it gives me 500 internal server error.
1- should I use Apache instead?
2- Can anyone share a working Nginx conf file with SSL?
3- is PHP 7.4.2 safe? or should I use an older version?
4- I am using Ubuntu 20.10 on Azure, should I go with 20.04?

Basically, I am trying to find out if this is a compatibility issue or just Nginx configuration.
Can anyone help?

**Comment by chriskapp** [comment]
Hi, so it is also be also possible to use Nginx instead of Apache. Recently a user has added the following Nginx s. https://fusio.readthedocs.io/en/latest/setup/nginx.html but this is without SSL. Regarding PHP we support PHP 7.3, 7.4 and 8.0 so you are free to chosse the fitting version. Ubuntu 20.10 should be fine, we also use Ubuntu in combination with Apache and Letsencrypt and everything works fine, if you use the certbot it will also automatically adjust your webserver config. But then you need to set also the base url of Fusio to `https`

**Comment by cococonuts** [comment]
I used let's encrypt to certificate nginx. But the I am not able to login https://antaresapi.com/apps/fusio/login.
Do you have document for config https fusio?
Where can I set base url of Fusio to https?

**Comment by cococonuts** [comment]
@chriskapp  Do you use Apache and Letsencrypt for your own website? Could you share your apache config?

**Comment by chriskapp** [comment]
@cococonuts sure so for my own site I use the Fusio docker container s. https://github.com/apioo/fusio-docker/blob/main/apache/fusio.conf and an Nginx reverse proxy with lets encrypt, so the communication between Fusio and Nginx is plain HTTP and only Nginx handles the SSL traffic.

**Comment by chriskapp** [comment]
Regarding nginx, we have a working nginx config at s. https://docs.fusio-project.org/docs/installation/nginx
